train.py

```diff
@@ -16,65 +17,60 @@ def train(data_cases):
 def recursive(data_cases):
     print("Begin2" +str(len(data_cases)))
```

Comment:
Can you run an auto format on this file? It's mainly single spaces between operator being off

Reply:
sure thing